### ISSUE TITLE
AttributeError: 'OneToOneRel' object has no attribute 'parent_model'

### ISSUE THREAD

**Issue by JoeReis** [ricomoss/django-fixtureless#23]
When I run my tests, I get this error. Recently upgraded to Django 1.8.

```======================================================================
ERROR: test_user_profile_list_template (user.tests.tests_urls.TestUrls)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/josephreis/Documents/Projects/sidehacker/user/tests/tests_urls.py", line 22, in setUp
    self.profile = factory.create(Profile)
  File "/Users/josephreis/.virtualenvs/sidehacker/lib/python3.4/site-packages/fixtureless/factory.py", line 70, in create
    return self._deliver(*args, save=True)
  File "/Users/josephreis/.virtualenvs/sidehacker/lib/python3.4/site-packages/fixtureless/factory.py", line 66, in _deliver
    objs = tuple(save_instances(pipeline) if kwargs['save'] else pipeline)
  File "/Users/josephreis/.virtualenvs/sidehacker/lib/python3.4/site-packages/fixtureless/factory.py", line 77, in save_instances
    for instance in iterable:
  File "/Users/josephreis/.virtualenvs/sidehacker/lib/python3.4/site-packages/fixtureless/factory.py", line 56, in <genexpr>
    for kwargs in kwargs_iter)
  File "/Users/josephreis/.virtualenvs/sidehacker/lib/python3.4/site-packages/fixtureless/generator.py", line 258, in create_instance
    isinstance(instance, field.related.parent_model)):
AttributeError: 'OneToOneRel' object has no attribute 'parent_model' 
```
Problem line:
```
tests_urls.py, line 22: self.profile = factory.create(Profile)
```

Which references the one-to-one field in Profile model:

```
user = models.OneToOneField(User)
```

**Comment by ricomoss** [comment]
I am aware of this issue.  The README file for django-fixtureless states that 1.8 is not yet supported but efforts are underway.  Django created an API for getting at this meta data and no longer supports the previous methods.

**Comment by JoeReis** [comment]
Sorry, didn't see the latest README. "pip install fixtureless -U" doesn't show me the latest README verbiage. Thanks and looking forward to the update.

**Comment by phretor** [comment]
bump!

**Comment by ricomoss** [comment]
Fixed as of version 1.4.3
